project.mml

```diff
@@ -518,6 +518,18 @@ Layer:
             WHEN access='no' AND (bicycle IS NULL OR bicycle='no') THEN 'no'
             ELSE NULL
           END AS can_bicycle,
+          CASE
+            WHEN bicycle IN ('designated') THEN 'yes'
+            ELSE NULL
+          END AS designated,
+          CASE
+            WHEN tags->'segregated' IN ('yes') THEN 'yes'
+            ELSE 'no'
+          END AS segregated,
+          CASE
+            WHEN tags->'oneway:bicycle' IN ('yes', '-1') THEN 'yes'
```

Comment:
`THEN tags->'oneway:bicycle'` to keep the direction, or we will never need it?

Reply:
I don't understand the question.